modules/floating-buttons/classes/render/contact-buttons-render-base.php

```diff
@@ -403,14 +403,18 @@ protected function is_url_link( string $platform ): bool {
 	protected function render_link_attributes( array $link, string $key ) {
 		switch ( $link['platform'] ) {
 			case Social_Network_Provider::WAZE:
-				if ( ! empty( $link['location']['url'] ) ) {
-					$this->widget->add_link_attributes( $key, $link['location'] );
+				if ( empty( $link['location']['url'] ) ) {
+					$link['location']['url'] = '#';
```

Comment:
@louiswol94 this is needed because otherwise the link renders without an href and this means that a CSS rule is triggered where the color becomes black. This solves another issue that was discussed yesterday as wontfix